src/__test__/components/plots/ContinuousEmbeddingPlot.test.jsx

```diff
@@ -101,6 +102,35 @@ describe('Continuous embedding plot', () => {
     expect(screen.getByRole('graphics-document', { name: 'Continuous embedding plot' })).toBeInTheDocument();
   });
 
+  it('Calls generateData with truncatedPlotData when config.truncatedValues is true', async () => {
+    // Spy on the function
+    const generateDataSpy = jest.spyOn(PlotSpecGenerators, 'generateData');
+
+    await renderContinuousEmbeddingPlot(storeState, {
+      config: {
+        ...initialPlotConfigStates.embeddingContinuous,
+        truncatedValues: true,
+      },
+    });
+
+    // Check if generateData was called with truncatedPlotData
+    expect(generateDataSpy).toHaveBeenCalledWith(
+      expect.anything(), // cellSets
+      expect.anything(), // selectedSample
+      truncatedPlotData, // truncatedPlotData
+      expect.anything(), // embeddingData
+    );
+
+    // Cleanup
+    generateDataSpy.mockRestore();
+  });
+
+  it('Renders correctly with truncated data', async () => {
```

Comment:
Do we need this test I think the other tests are doing the same check?

Reply:
you're right, actually it was supposed to be testing what got broken. Removed it as I added a better suited test already